src/components/Main/Team/Team.tsx

```diff
@@ -1,17 +1,19 @@
 import React, { useEffect, useState } from 'react';
 import Profile from './Profile';
 import { request } from 'graphql-request';
+import {Row, Col} from 'react-bootstrap';
+import { isMobile } from 'react-device-detect';
 
-// TODO: Make a profile component with photo, name, position
-// TODO: Add beginning copy
 // TODO: (can be separate PR), have an archive of old exec
 interface Props {
   id: string;
 }
 
 function Team(props: Props) {
-  const [team, setTeam] = useState([]);
-  
+  //const [team, setTeam] = useState([]);
+  const [team, setTeam] = useState<any[]>([]);
```

Comment:
don't use any, if you're going to declare a type, be specific, otherwise feel free to go back to previous syntax